Solutions/GitHub/Hunting Queries/Suspicious Fork Activity.yaml

```diff
@@ -0,0 +1,28 @@
+id: 467e6a70-adc9-49b7-8cf3-f21927c71159
+name: GitHub OAuth App Restrictions Disabled
+description: |
+  'This hunting query identifies a fork activity against a repository done by a user who is not the owner of the repo nor a contributes.'
+requiredDataConnectors: []
```

Comment:
Please add the connectorid and datatype in this format
requiredDataConnectors: -
connectorId: connector id
dataTypes: - name of the parser which we use in the query

Thanks,
Aditi Maheshwari

Reply:
Added